.github/workflows/deploy.devnet.yml

```diff
@@ -1,190 +0,0 @@
----
-name: DevNet Workflow
```

Comment:
We could probably use this workflow to spin up a new devnet on demand on AWS and then we could perform automatic or even manual testing against it. WDYT?